StreamChatIntegrationSource/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  StreamChatIntegration-SPM
+//
+//  Created by Boris Bielik on 16/11/2021.
+//
+
+import UIKit
+import StreamChatUI
+
+class ViewController: ChatChannelVC {
```

Comment:
You can remove this file, right?

Reply:
Thanks for spotting this. File is removed :)